src/jsutils/instanceOf.ts

```diff
@@ -10,7 +10,7 @@ export const instanceOf: (value: unknown, constructor: Constructor) => boolean =
   /* c8 ignore next 6 */
   // FIXME: https://github.com/graphql/graphql-js/issues/2317
   // eslint-disable-next-line no-undef
```

Comment:
looks like ci thinks this line can be deleted

Reply:
Will remove it and push the changes. Thanks for reviewing this, @yaacovCR!

Reply:
Done ✅